infrastructure/scripts/status.sh

```diff
@@ -16,19 +16,28 @@ wait-for-service startup-helper api-auth 80 10 api-auth
 
 CORE_ID=$(kubectl get configmap core-config -o jsonpath='{.data.CORE_ID}')
 API_HOSTNAME=$(kubectl get ingress airy-core -o jsonpath='{.spec.rules[0].host}')
-FACEBOOK_WEBHOOK_PUBLIC_URL="https://fb-${CORE_ID}.tunnel.airy.co"
-GOOGLE_WEBHOOK_PUBLIC_URL="https://gl-${CORE_ID}.tunnel.airy.co"
-TWILIO_WEBHOOK_PUBLIC_URL="https://tw-${CORE_ID}.tunnel.airy.co"
+NGROK_ENABLED=$(kubectl get configmap core-config -o jsonpath='{.data.NGROK_ENABLED}')
+
+if [ ${NGROK_ENABLED} = "true" ]; then
+    FACEBOOK_WEBHOOK_PUBLIC_URL="https://fb-${CORE_ID}.tunnel.airy.co/facebook"
+    GOOGLE_WEBHOOK_PUBLIC_URL="https://gl-${CORE_ID}.tunnel.airy.co/google"
+    TWILIO_WEBHOOK_PUBLIC_URL="https://tw-${CORE_ID}.tunnel.airy.co"
+else
+    WEBHOOKS_HOSTNAME=$(kubectl get configmap HOSTNAMES -o jsonpath='{.data.WEBHOOKS_HOST}')
+    FACEBOOK_WEBHOOK_PUBLIC_URL="https://${WEBHOOKS_HOSTNAME}/facebook"
+    GOOGLE_WEBHOOK_PUBLIC_URL="https://${WEBHOOKS_HOSTNAME}/google"
+    TWILIO_WEBHOOK_PUBLIC_URL="https://${WEBHOOKS_HOSTNAME}/twilio"
+fi
```

Comment:
The status script prints either the webhook configured in the ingress or the randomly generated Ngrok tunnel url.